idea_town/frontend/static-src/app/templates/header-view.js

```diff
@@ -10,7 +10,7 @@ export default `
           </header>
           <div class="town-background"></div>
           <div id="avatar-wrapper">
-            <p>Logged in as {{session}} <button data-hook="logout">Log out</button>
+            <i class="default-avatar" data-hook="logout"></i>
```

Comment:
bootstrap used to use `<i>` for icons, guess that was just stuck in my head. I guess `span` would be more semantic. What do you think?